Scripts/SwiftLint.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-SWIFTLINT="$(PWD)/Pods/SwiftLint/swiftlint"
-CONFIG="$(PWD)/.swiftlint.yml"
+if [ -z "$PROJECT_DIR" ]; then
+	pushd "$(dirname $0)/.." > /dev/null
+	PROJECT_DIR=`pwd`
+	popd > /dev/null
```

Comment:
If you wish to use some bash foo, and still keep the `PROJECT_DIR` variable, you might want to use something like: `PROJECT_DIR=${PROJECT_DIR:-$(dirname $0)/..}`

Reply:
I actually initially tried `$(dirname $0)/..`, but swiftlint crashes on that (can't read the config file). This is the first thing I found that normalizes the path, that isn't some massive function or depends on python (or something else).

Reply:
1. If that's an issue, there's always the ability to explicitly pass to SwiftLint the exact path to the `.swiftlint.yml` file
2. Otherwise, if SwiftLint can't read the config file when you use `$(dirname $0)/..` directly, that means that it will likely crash in other situations depending on how you invoke that script, because after all if `PROJECT_DIR` isn't initially set, then your script will set it to the same value as `$(dirname $0)/..` anyway, probably leading to the same behavior?

The fact that `popd` exits with error if you pass to `pushd` a path that points exactly to the current directory (which means it won't push anything so the directory stack will still be empty and `popd` will error) makes me think that's not the best solution either anyway.

Reply:
Another simpler option would be to `cd $(dirname $0)/..` at the beginning of the script and `cd -` back at the end of the script. If we do that though, we'd just have to be careful to put that `cd -` not just at the end of the script, but also on interruption or error during shell (`trap … EXIT`)

```bash
cd $(dirname $0)/..
function restore_dir {
  cd -
}
trap restore_dir EXIT

… rest of the script …
```